Unreal/CarlaUE4/Plugins/CarlaTools/Source/CarlaTools/Private/OpenDriveToMap.cpp

```diff
@@ -266,7 +268,7 @@ void UOpenDriveToMap::CreateTerrainMesh(const int MeshIndex, const FVector2D Off
 
 AActor* UOpenDriveToMap::SpawnActorWithCheckNoCollisions(UClass* ActorClassToSpawn, FTransform Transform)
 {
-  UWorld* World = UEditorLevelLibrary::GetEditorWorld();
+  UWorld* World = GEditor->GetEditorWorldContext().World();
```

Comment:
**Code Clean***
The next code should be also removed from this file as this include is not used anymore:
```c++
#include "EditorLevelLibrary.h"
```